.sync.yml

```diff
@@ -31,7 +31,7 @@
     stage: acceptance
   - bundler_args: 
     dist: trusty
-    env: PLATFORMS=debian_puppet6
+    env: PLATFORMS=deb_puppet6
     rvm: 2.5.3
     before_script:
     - bundle exec rake 'litmus:provision_list[waffle_deb]'
```

Comment:
Hey @david22swan ,

This should be updated to travis_deb.

Cheers!

Reply:
fixed